ruby/bob.rb

```diff
@@ -0,0 +1,37 @@
+class Bob
+  def hey(remark)
+    @remark = Remark.new(remark)
+    return "Fine. Be that way!" if @remark.is_blank?
+    return "Whoa, chill out!" if @remark.is_shouting?
+    return "Sure." if @remark.is_question?
```

Comment:
It seems you are including multiple returns, can you format this to only return once?